int16 pos_x
int16 pos_y
int16 pos_theta
int16 v_x
int16 v_y
int16 v_theta
int16 bola_x
int16 bola_y